subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheIntegrationTest.groovy

```diff
@@ -1114,4 +1115,45 @@ class ConfigurationCacheIntegrationTest extends AbstractConfigurationCacheIntegr
         then:
         outputContains("value = value")
     }
+
+    @Issue("gradle/gradle#14465")
+    def "configuration is cacheable when providers are used in settings"() {
```

Comment:
:x: Those tests would be better in `ConfigurationCacheBuildOptionsIntegrationTest`

Reply:
Moved.